app/app.go

```diff
@@ -808,6 +839,13 @@ func NewApplication(
 			ctx.Logger().Info("start to run pstake migration...")
 
 			//add more upgrade instructions
+			ctx.Logger().Info("Running revert of tombstoning")
+			err := upgrades.RevertCosTombstoning(ctx, app.SlashingKeeper, app.MintKeeper, &bankkeeper.BaseKeeper{}, app.StakingKeeper)
+			if err != nil {
+				panic(fmt.Sprintf("failed to revert tombstoning: %s", err))
+			}
+
+			// RunMigrations twice is just a way to make auth module's migrates after staking
```

Comment:
remove this comment, it was needed only for v0.42-0.44.. auth vesting account fix.